performance-tests/src/scenario-02/s02.test.ts

```diff
@@ -54,7 +56,8 @@ export function setup(): IdentityWithToken[] {
     const testIdentities: IdentityWithToken[] = [];
 
     for (let i = 0; i < scenario.preAllocatedVUs; i++) {
-        const { httpResponse, generatedPassword } = createIdentity(client, "test", "test");
+        const password = "test-password";
+        const httpResponse = createIdentity(client, "test", "test", password);
```

Comment:
I think we should extract these into global constants. You can put them into the same file as the api version

Reply:
Ah, and by "these" I mean password, clientId and clientSecret.